### DISCUSSION TITLE
Linux agent file format?

### DISCUSSION THREAD

**Discussion by mstrkoffee** [Security-Onion-Solutions/securityonion#11233] [2.4] [answered]
### Version

2.4.100

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

48

### Storage for /

5TB

### Storage for /nsm

4.8 TB

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

When downloading the linux elastic agent it shows up as so-elastic-agent_linux_amd64 with no file extension listed.  The file size is 310972236 - my forensic skills arent helping me much. 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by kvkamin** [comment, marked as answer]
you will have change permissions as privileged user , ie
chmod +x so-elastic-agent_linux_amd64
to install - as privileged user:
./so-elastic-agent_linux_amd64
